dcsazure_AzureSQL_to_AzureSQL_mask_pl/dcsazure_AzureSQL_to_AzureSQL_mask_pl.json

```diff
@@ -0,0 +1,3422 @@
+                          }
+                        ],
+                        "policy": {
+                          "timeout": "0.12:00:00",
+                          "retry": 0,
+                          "retryIntervalInSeconds": 30,
+                          "secureOutput": false,
+                          "secureInput": false
+                        },
+                        "userProperties": [],
+                        "typeProperties": {
+                          "dataflow": {
+                            "referenceName": "dcsazure_AzureSQL_to_AzureSQL_filtered_mask_df",
+                            "type": "DataFlowReference",
+                            "parameters": {
+                              "runId": {
+                                "value": "'@{pipeline().RunId}'",
+                                "type": "Expression"
+                              },
+                              "DF_SOURCE_SCHEMA": {
+                                "value": "'@{item().source_schema}'",
+                                "type": "Expression"
+                              },
+                              "DF_SOURCE_TABLE": {
+                                "value": "'@{item().source_table}'",
+                                "type": "Expression"
+                              },
+                              "DF_SINK_SCHEMA": {
+                                "value": "'@{item().sink_schema}'",
+                                "type": "Expression"
+                              },
+                              "DF_SINK_TABLE": {
+                                "value": "'@{item().sink_table}'",
+                                "type": "Expression"
+                              },
+                              "DF_FIELD_ALGORITHM_ASSIGNMENT": {
+                                "value": "'@{activity('Get Masking Parameters With Filter').output.runStatus.output.MaskingParameterOutput.value[0].FieldAlgorithmAssignments}'",
+                                "type": "Expression"
+                              },
+                              "DF_COLUMNS_TO_MASK": {
+                                "value": "@activity('Get Masking Parameters With Filter').output.runStatus.output.MaskingParameterOutput.value[0].ColumnsToMask",
+                                "type": "Expression"
+                              },
+                              "DF_BODY_TYPE_MAPPING": {
+                                "value": "@activity('Get Masking Parameters With Filter').output.runStatus.output.MaskingParameterOutput.value[0].DataFactoryTypeMapping",
+                                "type": "Expression"
+                              },
+                              "DF_NUMBER_OF_BATCHES": {
+                                "value": "@activity('Get Masking Parameters With Filter').output.runStatus.output.MaskingParameterOutput.value[0].NumberOfBatches",
+                                "type": "Expression"
+                              },
```

Comment:
You will need to change the parameter from DF_NUMBER_OF_BATCHES to DF_NUMBER_OF_ROWS_PER_BATCH (default value of 1000) - and update the calls to this dataflow (filtered_mask) to use the variable TARGET_BATCH_SIZE which should have a default value of 2000